justfile

```diff
@@ -13,4 +13,6 @@ docs-config := base-directory / "docs/mkdocs.yml"
 
 # Serve current docs located in ./docs/docs
 serve-docs port="8000":
+    typer {{base-directory}}/kpops/cli/main.py utils docs --name kpops --output {{base-directory}}/docs/docs/user/references/cli-commands.md
```

Comment:
Besides, this should not be under the `serve-docs` command. Please put it in separate command

Reply:
done